frontend/src/components/Search/SuggestedCategories/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from "@mui/material"
+
+export const SuggestedCategories = ({options, onClick}: {options: string[], onClick: () => void}) => {
+  if (options.length === 0) return <></>
+
+  return (
+    <div className="pt-2">
+      We found suggested categories for your query: <br /> {options.join(', ')} <br />
```

Comment:
do we really need to have some text? I thought the categories would be just automatically assigned, even though the user has selected some of them before.